testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -441,8 +443,8 @@ protected SqlOperatorFixture fixture() {
   }
 
   /** Generates parameters to test both regular and safe cast. */
-  static Stream<Arguments> safeParameters() {
-    SqlOperatorFixture f = SqlOperatorFixtureImpl.DEFAULT;
+  private Stream<Arguments> safeParameters() {
```

Comment:
We remove static here to allow use of non-static method `fixture()` to use subclass's overridden implementation, and this needs to add `@TestInstance(TestInstance.Lifecycle.PER_CLASS)` to this test class to allow this. See more at https://github.com/junit-team/junit5/issues/984